src/components/atomic/molecules/Navigation.tsx

```diff
@@ -7,17 +7,27 @@ import { Link, useLocation } from "react-router-dom";
 import { Menu, Space } from "antd";
 import { NETWORKS, SUPPORTED_NETWORKS } from "../../../config/network";
 import { RiSafe2Line } from "react-icons/ri";
-import { useBreakpoints, useChainId, useRequestChangeNetworkCallback, useTranslator } from "hooks";
+import {
+  useBreakpoints,
+  useChainId,
+  useRequestChangeNetworkCallback,
+  useTranslator,
+} from "hooks";
 import { useMemo } from "react";
 import Icon from "@ant-design/icons";
 
 const NETWORKS_BY_ID: Record<number, any> = {
-  '1': NETWORKS.mainnet,
-  '137': NETWORKS.polygon
-}
+  "1": NETWORKS.mainnet,
```

Comment:
JSPerf is supposedly defunct now, but it seems like it'd be more performant to access object properties with keys that are integers vs. strings: https://stackoverflow.com/a/10639526.

Reply:
Very interesting indeed @kphed, will take note of this for the upcoming refactoring.